.github/actions/sls-remove/action.yml

```diff
@@ -49,7 +49,14 @@ runs:
     - name: Remove CF Stack
       if: ${{ env.SLS_STAGE != 'production' && env.SLS_STAGE != 'dev' }}
       shell: bash
-      run: yarn workspace @asap-hub/${{ inputs.app }} run pnpify sls remove --verbose
+      run: |
+        set +e
```

Comment:
This is a bit mysterious if you haven't recently spent time reading the bash documentation for the `-e` flag. I'd be tempted to add an explaining comment.